recipes/four0four.md

```diff
@@ -22,4 +22,8 @@ module.exports = {
   ...
 };
 
-```
\ No newline at end of file
+```
+
+### 🍰 Dessert
+
+You deserve some cookies (or biscuits or whatever they're call in your neck of the woods).
```

Comment:
Ah there's a spelling mistake in every recipe with this sentence, "call" -> "called". Not really worth changing since it's in multiple recipes but thought I should note that for future reference 😅